amazon-sqs/5.9/modules/ROOT/pages/amazon-sqs-connector-config-topics.adoc

```diff
@@ -0,0 +1,97 @@
+= Amazon SQS Connector 5.9 Additional Configuration Information - Mule 4
+:page-aliases: connectors::amazon/amazon-sqs-connector-config-topics.adoc
+
+== Long and Short Polling
+
+Since version 5.9, the default behavior of the Receive Messages source and the Read operation is to use long polling:
+
+* With long polling, SQS Connector returns the query results as soon as possible, if any messages are present.
++
+Otherwise, SQS Connector waits the specified amount of seconds before returning an empty response.
+* With short polling, SQS Connector always returns the query results as soon as possible, even if that means returning an empty response.
+
+Long polling can reduce the number of network requests without affecting the throughput.
+
+You can control polling behavior either globally or at the individual source and operation level.
```

Comment:
```suggestion
You can configure SQS Connector to control polling behavior either globally or locally, and operation level.
```

Reply:
Though, at the operation level is not explained later in the next section. Maybe you can remove it, because the explanation mentions only the Global Configuration Element of Studio.